src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -269,6 +269,18 @@ else if (flutterLaunchMode == FlutterLaunchMode.RELEASE) {
     return new FlutterCommand(this, root.getRoot(), FlutterCommand.Type.ATTACH, args.toArray(new String[]{ }));
   }
 
+  public FlutterCommand flutterRunWeb(@NotNull PubRoot root, @NotNull RunMode mode) {
+    // flutter packages pub global run webdev serve [--debug] [--hot-reload]
+    final List<String> args = new ArrayList<>();
+    args.add("global");
+    args.add("run");
+    args.add("webdev");
+    args.add("daemon");
+    // TODO After debug is supported by webdev, this should be modified to check for debug and add any additional needed flags:
```

Comment:
add a bug related to this TODO or add your username to the todo.

Reply:
done, https://github.com/flutter/flutter-intellij/issues/3349